src/com/android/settings/wifi/repository/WifiHotspotRepository.java

```diff
@@ -384,11 +384,7 @@ public boolean isDualBand() {
      * @return {@code true} if Wi-Fi 5 GHz Band is supported
      */
     public boolean is5GHzBandSupported() {
-        if (mIs5gBandSupported == null) {
-            mIs5gBandSupported = mWifiManager.is5GHzBandSupported();
```

Comment:
Why are is5GHzBandSupported() and is6GHzBandSupported() checks removed?

Reply:
Because that's the whole point of this PR and feature request? Google has been ignoring for years that the Linux kernel specifies normal frequencies, and Google is limiting them for no reason